recipes/dmbiolib/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "dmbiolib" %}
+{% set version = "0.3.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 26d2974bbe19c2af337e364b1b93f902300fd88d8635b02af2467288a6d9cde9
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3
+  run:
+    - python >=3
+    - numpy
+    - matplotlib-base
+
+test:
+  commands:
+    - python -m dmbiolib -v
+
+about:
+  home: "https://github.com/damienmarsic/dmbiolib"
+  license: "GNU General Public v3 (GPLv3)"
+  license_family: GPL3
+  license_file: LICENSE
+  summary: "Library of Python functions for bioinformatics"
+  doc_url: https//dmbiolib.readthedocs.io
+  dev_url: 
```

Comment:
remove key?

Reply:
Thanks for the suggestions. I made the requested changes.